lib/diagnostic.rb

```diff
@@ -106,10 +106,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = puts "#{person[:first_name]} #{person[:age]}"
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = puts "My name is #{person[:first_name]}"
```

Comment:
Assuming that `person` is defined, this will print to the console but not set it as the default return value to `person`.  It should be `person.default = "My name is #{person[:first_name]}"`.